File: comms/push-notifications/users/delete-user/delete-user.cs
using Twilio.Comms;
using System.Threading.Tasks;

public partial class Examples
{
    public async Task Example() {
        var client = new TwilioComms(
            accountId: "TWILIO_ACCOUNT_SID",
            authToken: "TWILIO_AUTH_TOKEN"
        );

        await client.PushNotifications.Users.DeleteAsync(
            "comms_pushnotificationuser_01h9krwprkeee8fzqspvwy6nq8"
        );
    }

}

File: comms/push-notifications/users/delete-user/delete-user.py
from twilio_comms import TwilioComms

client = TwilioComms(
    account_id="TWILIO_ACCOUNT_SID",
    auth_token="TWILIO_AUTH_TOKEN",
)

client.push_notifications.users.delete(
    user_id="comms_pushnotificationuser_01h9krwprkeee8fzqspvwy6nq8",
)
